core/src/test/java/io/grpc/internal/AbstractManagedChannelImplBuilderTest.java

```diff
@@ -32,11 +32,19 @@
 package io.grpc.internal;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.fail;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import io.grpc.Attributes;
+import io.grpc.NameResolver;
 
 import java.net.InetSocketAddress;
 import java.net.URI;
 import java.util.concurrent.TimeUnit;
+
```

Comment:
Creating extra sections here is the old style. Now there are only two sections: static imports and normal imports.